src/mpyl/cli/__init__.py

```diff
@@ -34,7 +34,7 @@ class CliContext:
 
 @dataclass(frozen=True)
 class MpylCliParameters:
-    local: bool = False
+    local: bool = True
```

Comment:
This is a bit dangerous because now we need to check in every runner if we set the local param. It was implicitly assumed to be set to False before

Reply:
https://github.com/Vandebron/mpyl/pull/233/commits/ee593e72b9871535bf76b18cdc01ba80554a2251